Tests/scripts/test_modeling_rules.sh

```diff
@@ -24,10 +23,34 @@ for modeling_rule in "${MODELING_RULES_ARRAY[@]}"; do
   fi
 done
 
-if [[ -z "$MODELING_RULES_TO_TEST" ]]; then
-    echo "There was a problem reading the list of modeling rules that require testing from '$ARTIFACTS_FOLDER/modeling_rules_to_test.txt'"
+if [[ -z "${MODELING_RULES_TO_TEST}" ]]; then
+    echo "There was a problem reading the list of modeling rules that require testing from '${ARTIFACTS_FOLDER}/modeling_rules_to_test.txt'"
     exit 1
 fi
 
-echo "Testing Modeling Rules"
-demisto-sdk modeling-rules test --xsiam-url="$XSIAM_URL" --auth-id="$AUTH_ID" --api-key="$API_KEY" --xsiam-token="$XSIAM_TOKEN" --non-interactive $(echo "$MODELING_RULES_TO_TEST")
+if ! DEMISTO_SDK_SKIP_VERSION_CHECK=True demisto-sdk modeling-rules test --help 2>&1 | grep 'junit-path'; then
+  MODELING_RULES_RESULTS_FILE_NAME="${ARTIFACTS_FOLDER}/modeling_rules_results.xml"
+  MODELING_RULES_RESULTS_ARG=(" --junit-path" "${MODELING_RULES_RESULTS_FILE_NAME}")
+  echo "Testing Modeling Rules - Results will be saved to ${MODELING_RULES_RESULTS_FILE_NAME}"
+else
+  echo "Testing Modeling Rules - demisto-sdk version is too old, skipping junit report generation"
+  MODELING_RULES_RESULTS_ARG=()
+  MODELING_RULES_RESULTS_FILE_NAME=""
+fi
+
+demisto-sdk modeling-rules test --xsiam-url="$XSIAM_URL" --auth-id="$AUTH_ID" --api-key="$API_KEY" \
+  --xsiam-token="$XSIAM_TOKEN" --non-interactive "${MODELING_RULES_RESULTS_ARG[@]}" \
+  ${MODELING_RULES_TO_TEST}
+exit_code=$?
+
```

Comment:
Or add it to an `else` of the `if` below.
```suggestion
  echo "Finished testing Modeling Rules, exit_code=$exit_code"

```